saleor/static/dashboard/scss/components/_card-collapse.scss

```diff
@@ -1,5 +1,4 @@
 .collapse {
-  overflow: hidden;
   max-height: 1000px;
```

Comment:
It's not as simple as that. Please notice that `overflow: hidden` is used to trim content in responsive mode, therefore this property shouldn't be simply removed. I'd wrap it in a `@media (max-width: 992px)` to completely shut it down on desktop-sized displays and try disabling it while filter bar is expanded, using `transition-delay: $transition-duration`. Good luck :)

Reply:
I gonna try to have a look into that! I saw your commit before doing this PR using git blame but didn't find any breaking. I'm gonna look what I can do.

Reply:
After a look, I see the issue, I didn't see it when trying on desktop yesterday. I'm suggesting to move the overflow hidden to `.collapse.collapsed`:
```css
.collapse.collapsed {
    overflow: hidden;
}
```

I'll be waiting to hear back from you.

Reply:
Okay, it looks cool - I thought that maybe filter bar's content would be visible while animation was running, but it works okay.